back/app/models/verification/verification_method.rb

```diff
@@ -25,5 +25,9 @@ def config
     def enabled_for_verified_actions?
       config[:enabled_for_verified_actions] || false
     end
+
+    def enabled?
+      true
+    end
```

Comment:
Needed because for keycloak, the verification part is turned off for Rheinbahn - but it still needs all the verification config etc. Will become less confusing when I finally merge my refactor.